File: 26_Remove_Duplicates_From_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count = 0;
        for(int i = 1; i < nums.size(); i++){
			// Count repeat
            if(nums[i] == nums[i-1]) count++;
			// Overwrite repeat
            else nums[i-count] = nums[i];
        }
        return nums.size()-count;
    }
};
// Final vector will have all unique member and bunch of empty space at the end

File: language/python/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        j = 1
        first = nums[0]
        last = nums[-1]
        while first != last:
            if nums[j] > first:
                first = nums[j]
                j += 1
            else:
                nums.pop(j)
        return j
